src/app/reviews/reviews.component.html

```diff
@@ -0,0 +1,23 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title center>Reviews</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content padding color="light">
+  <div class="top-round-btns" text-center>
+    <ion-button class="do" [ngClass]="{'active-review': activeToggle}" (click)="toggleActiveDo()">Todo</ion-button>
+    <ion-button class="done" [ngClass]="{'active-review': !activeToggle}" (click)="toggleActiveDone()">Done</ion-button>
```

Comment:
After click, we should disable this button. 
The problem of no doing that is currently if you click "Todo", and click "Todo" again, the button style changed, but the content doesn't

Reply:
Also, just a suggestion. 
"Toggle" usually means we can turn it on and off. Maybe consider change the name to "show" or "display" since the only behaviour we allow for clicking this button is to "display/show" todo/done page. 
Or if it is only one button, calling it "toggle" make sense that you can switch between two status